Sources/Modifiers.swift

```diff
@@ -649,3 +649,80 @@ private final class PointerOfImpl<T : Arbitrary> : Arbitrary {
 	}
 }
 
+/**
+ Class used to generate values from mulitple `Gen` instances.
+ 
+ Given a StdGen and size, generate values from other generators, splitting the StdGen
+ after each call to `generate`, ensuring suffiicent entropy across generators.
+ 
+ - seealso: Gen.compose
+*/
+public final class GenComposer {
+    private var stdgen: StdGen
+    private var size: Int
+    
+    init(stdgen: StdGen, size: Int) {
+        self.stdgen = stdgen
+        self.size = size
+    }
+    
+    // Split internal StdGen to ensure sufficient entropy over multiple `generate` calls.
+    private func split() -> StdGen {
+        let old = stdgen
+        stdgen = old.split.0
+        return old
+    }
+    
+    /// Generate a new `T` with a specific generator.
+    ///
+    /// - parameter gen: The generator used to create a random value.
+    /// 
+    /// - returns: A random `T` using the receiver's stdgen and size.
+    ///
+    public func generate<T>(gen: Gen<T>) -> T {
+        return gen.unGen(split(), size)
+    }
+    
+    ///  Generate a new `T` with it's default `arbitrary` generator.
+    ///
+    ///  - returns: A random `T`.
+    ///
+    ///  - seealso: generate\<T\>(gen:)
+    public func generate<T where T: Arbitrary>() -> T {
+        return generate(T.arbitrary)
+    }
+}
+
+extension Gen {
+    /**
```

Comment:
`///` style comments are the one true way going forward. Please unify the comment style.